lxd/certificates.go

```diff
@@ -16,24 +16,20 @@ import (
 	"github.com/lxc/lxd/shared"
 )
 
+func certGenerateFingerprint(cert *x509.Certificate) string {
+	return fmt.Sprintf("%x", cert.SerialNumber)
```

Comment:
The Serial is an identifier that can be set to any value by the user at certificate creation time, it's therefore unsuitable as a unique identifier since it'd be trivial to forge a certificate that'd then look identical.